### PR TITLE
CivicActions team default incident response and contingency plans

### PR DESCRIPTION
**Pull request by openprivacy**
These are the default contingency and incident response plans that all CivicActions team members should be aware of. They are based on plans currently in use by the technology operations teams for two federal clients.

### CODE REVIEW

File: docs/09-security/contingency-plan.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+## Overview
+
+This Contingency Plan provides baseline guidance for the CivicActions Team when managing the disruption, compromise, or failure of any component of a CivicActions maintained system, product or service ("system"). As a general guideline, we consider "disruption" to mean unexpected downtime or significantly reduced service lasting longer than:
+* 30 minutes 0900 - 2100 Eastern Time Monday through Friday (standard U.S. business hours)
+* 90 minutes at other times
+
+Scenarios where that could happen include unexpected downtime of key services, system data loss, or improper privilege escalation. In the case of a security incident, the team uses the [Security Incident Response Procedure](incident-response-plan.md) as well.
+
+Note that some clients will have created their own Contingency Plan defining procedures specfic to their system. In such a case, that Contingency Plan takes precedence.
```
Comment (kevwalsh) [inline docs/09-security/contingency-plan.md]: typo... "specific"

**Review by kevwalsh — COMMENTED** [review]
I'm a bit concerned about the blanket nature of "any component of a CivicActions maintained system, product, or service". 

We are involved with maintaining a couple handfuls of projects where i can't see this policy applying at the moment. I wonder if this policy (or set of policies) can be articulated as a service or set of services that specific projects formally adopt, rather than applying it as a blanket. 

Also, in my (admittedly not too close) read of this, it seems like much of Incidence Response is not specifically security, and many incidences (hopefully most!) are not security related.  I imagine that you're writing this in the context of security because security protocols require Incidence Response plans. But i'm wondering if this is the right home. That said, i'm not sure where it belongs. Engineering? Seems too specific.  Policies? Seems too company-wide.

File: docs/09-security/contingency-plan.md (hunk 1 above)
Comment (openprivacy) [reply docs/09-security/contingency-plan.md]: The incident response plan is created for compliance to have a policy for "security incidents" and so it possibly should be renamed. Non-security incidents should be similarly handled in general, but do not specifically fall under this compliance requirement (nor this plan).

The plan is supposed to be general and apply to any system we maintain should a security incident occur, and all CivicActions team members should be trained in the basic procedures (the Operations team comprised of Tech Leads and IT_IS) is required to be trained).

Does this understanding change your view? What projects do you see falling outside the scope of this plan (when constrained to security incidents)?

Comment (openprivacy) [reply docs/09-security/contingency-plan.md]: @marctjones would you be willing to review the incident response plan here and the comments above by @kev and myself and suggest changes before I make the next revision? Goal is to have a baseline security incident response process that everyone is exposed to and IT/Operations is trained in. This is based on https://cloud.gov/docs/ops/security-ir/ (another change to make: I will add an acknowledgement to the cloud.gov documentation). Thanks!

**Comment by openprivacy** [comment]
@marctjones can you take a look at the PR, specifically the first section of each doc ("Applicability") which reads (e.g.):
> **Note:**  This Incident Response Plan applies only to systems for which CivicActions has negotiated and defined Incident Response/Contingency Plan (IRCP) operations. Each IRCP-managed system will have a specific, tailored version of this Incident Response Plan or in some cases a completely unique Incident Response Plan will be developed. All CivicActions employees are aware of the procedures outlined herein.

**Comment by openprivacy** [comment]
@marctjones -- bump?

**Review by marctjones — APPROVED** [review]

**Pull request merged**